ui/raidboss/data/07-dt/raid/r12n.ts

```diff
@@ -0,0 +1,508 @@
+      durationSeconds: 6,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Away from snakes, platform breaking',
+        },
+      },
+    },
+    {
+      id: 'R12N Tiles Phase Wyrm Head Cleaves',
+      type: 'ActorControlExtra',
+      // PlayActionTimeline, shows the "head rises from the void" animation
+      netRegex: { category: '0197', param1: '11D3', capture: true },
+      preRun: (data, matches) => {
+        data.tilePhaseHeadIds.push(matches.id);
+      },
+      durationSeconds: 7,
+      infoText: (data, _matches, output) => {
+        if (data.tilePhaseHeadIds.length < 2)
+          return;
+        const [actor1Id, actor2Id] = data.tilePhaseHeadIds;
+
+        if (actor1Id === undefined || actor2Id === undefined)
+          return;
+
+        const [actor1, actor2] = [data.actorPositions[actor1Id], data.actorPositions[actor2Id]];
+
+        if (actor1 === undefined || actor2 === undefined)
+          return;
+
+        const head1Pos = getTileHeadPos(actor1);
+        const head2Pos = getTileHeadPos(actor2);
+
+        if (head1Pos.startsWith('out') && head2Pos.startsWith('out'))
+          return output.middle!();
+
+        if (head1Pos.endsWith('E') && head2Pos.endsWith('E'))
+          return output.west!();
+
+        if (head1Pos.endsWith('W') && head2Pos.endsWith('W'))
+          return output.east!();
+
+        const positions: TileHeadPos[] = [...tileHeadPositions].filter((pos) =>
+          pos !== head1Pos && pos !== head2Pos
+        );
+
+        const [safePos1, safePos2] = positions;
+
+        return output.text!({
+          dir1: output[safePos1 ?? 'unknown']!(),
+          dir2: output[safePos2 ?? 'unknown']!(),
+        });
+      },
+      run: (data) => {
+        if (data.tilePhaseHeadIds.length < 2)
+          return;
+        data.tilePhaseHeadIds = [];
+      },
+      outputStrings: {
+        middle: Outputs.middle,
+        east: Outputs.getRightAndEast,
+        west: Outputs.getLeftAndWest,
+        outW: {
+          en: 'Out West',
+        },
+        inW: {
+          en: 'In West',
+        },
+        inE: {
+          en: 'In East',
+        },
+        outE: {
+          en: 'Out East',
+        },
+        text: {
+          en: '${dir1}/${dir2}',
+        },
+      },
+    },
+    {
+      id: 'R12N Mindless Flesh',
+      type: 'StartsUsing',
+      netRegex: {
+        id: ['BBD8', 'BBD9', 'BBDA', 'BBDB', 'BBDC', 'BBDD', 'BBDE'],
+        source: 'Lindwurm',
+        capture: false,
+      },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Dodge lines',
+        },
+      },
+    },
+    {
+      id: 'R12N Dramatic Lysis',
+      type: 'StartsUsing',
+      netRegex: { id: 'BCF3', source: 'Lindwurm', capture: false },
+      suppressSeconds: 1,
+      response: Responses.spread(),
```

Comment:
This appears to be a duplicate with `Dramatic Lysis Spread` at line 212, and I did see duplicate calls in part 2. Is this an artifact or does it work around something else?

Reply:
Yeah, as you saw the `StartsUsing` trigger didn't work in P1, so I added the `HeadMarker` trigger instead and forgot to remove this one.